packages/cli/tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/index.ts", "src/mud.ts", "src/mud2.ts"],
+  target: "esnext",
+  format: ["esm"],
+  dts: false,
```

Comment:
is dts not needed when doing tsup in this way? or is it because cli doesn't have anything to import?

Reply:
from @dk1a 
> I don't think we do, since package.json now has "types": "src/index.ts", so the sources themselves replaces dts